openqa-maintenance.py

```diff
@@ -1029,6 +1026,10 @@ def setup_checker(self):
         return bot
 
 if __name__ == "__main__":
+    requests_log = logging.getLogger("requests.packages.urllib3")
+    requests_log.setLevel(logging.WARNING)
+    requests_log.propagate = False
+
```

Comment:
urllib3 already leverages it - and this is stackoverflow's recommendation how to shut it up.